src/main.cpp

```diff
@@ -838,9 +838,15 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
     {
         if (pool.mapNextTx.count(txin.prevout))
         {
+            if (!nRbfPolicy)
+                // No replacement allowed by policy
+                return state.Invalid(false, REJECT_CONFLICT, "txn-mempool-conflict");
+
             const CTransaction *ptxConflicting = pool.mapNextTx[txin.prevout].ptx;
-            if (!setConflicts.count(ptxConflicting->GetHash()))
-            {
+            if (setConflicts.count(ptxConflicting->GetHash()))
+                continue;
```

Comment:
There's never a strict need for a continue statement:
```
while(fCondition)
    if (!fA)
       continue;
    // Do AAA
}
```
Is equivalent to
```
while(fCondition)
    if (fA) {
      // Do AAA
    }
}
```
which is what we had and you are changing for no good reason.
The second version (what we currently have) is not only more readable but also more flexible, for example, say we want to change the code to do the following:

```
while(fCondition)
    if (fA) {
      // Do AAA
    }
   // Do BBB
}
```
In the second example it's just adding the new code out of the if statement, in the first case we have to rewrite the code like the first case and indent first.
The only reason I can think of for using is reducing this PR's total diff(by avoiding the indent to the code under the new condition), but it doesn't make the diff (or the final resulting code) more readable, so I really don't see the point.
I would generally avoid using continue unless it's one of those the rare cases where its use actually increases readability and reduces the potential for bug introduction (instead of the opposite as it usually does). 
For example, many continue statements in the same loop may be more readable than excessive nesting:


```
while(fCondition)
    if (!fA)
       continue;
    // Do AAA
    if (!fB)
       continue;
    // Do BBB
    if (!fC)
       continue;
    // Do CCC
    if (!fD)
       continue;
    // Do DDD
    // E, F, G...
    // Do ZZZ
}
```
is more readable than, but equivalent to
```
while(fCondition)
    if (fA) {
      // Do AAA
      if (fB) {
         // Do BBB
         if (fC) {
           // Do CCC
           if (fD) {
              // Do DDD
                // E, F, G...
    } // D
    } // C
    } // B
    } // A
    // Do ZZZ
}
```
Of course, the readability disadvantages of excessive nesting can always be fixed with sub-functions instead:

```
bool DoFromA()
{
   if(fA) {
       // Do AAA
       return DoFromB();
   }
   return false
}

while(fCondition)
    if (DoFromA()) {
      // Do ZZZ
    }
```